ga4gh/repo/rnaseq2ga.py

```diff
@@ -1,37 +1,25 @@
-"""
-Script to parse the output file produced by cufflinks.
-"""
 from __future__ import division
 from __future__ import print_function
 from __future__ import unicode_literals
 
-import os
 import sqlite3
-import argparse
 
-import utils
+import ga4gh.exceptions as exceptions
 
-utils.ga4ghImportGlue()
 
-import ga4gh.datarepo as datarepo  # NOQA
-
-
-def data_repo(path):
-    dataRepository = datarepo.SqlDataRepository(path)
-    dataRepository.open(datarepo.MODE_READ)
-    return dataRepository
+SUPPORTED_RNA_INPUT_FORMATS = ["cufflinks", "kallisto", "rsem"]
 
 
 class RNASqliteStore(object):
```

Comment:
@saupchurch just fyi we try and camel-case acronyms in variable names (so it's easier to tell where one word ends and the other begins); this should be `RnaSqliteStore`; I'll make the change (and other similar ones) in my branch